src/Physics/GlashowResonance/XSection/GLRESPXSec.cxx

```diff
@@ -94,9 +164,27 @@ bool GLRESPXSec::ValidProcess(const Interaction* interaction) const
   return true;
 }
 //____________________________________________________________________________
-bool GLRESPXSec::ValidKinematics(const Interaction * /*in*/) const
+void GLRESPXSec::Configure(const Registry & config)
 {
-  return true;
+
+  Algorithm::Configure(config);
+  this->LoadConfig();
+}
+//____________________________________________________________________________
+void GLRESPXSec::Configure(string config)
+{
+  Algorithm::Configure(config);
+  this->LoadConfig();
 }
 //____________________________________________________________________________
+void GLRESPXSec::LoadConfig(void)
+{
+
+  //-- load the differential cross section integrator
+  fXSecIntegrator = dynamic_cast<const XSecIntegratorI *> (this->SubAlg("XSec-Integrator"));
+  assert(fXSecIntegrator);
+
+  GetParam( "Wmin", fWmin ) ;
```

Comment:
Can we make the change of this Wmin parameter more specific? Suggestion is GLRES-Wmin we have other Wmin around the code and if would be much better if we could distinguish them easily